src/components/Dist/Dist.tsx

```diff
@@ -1,56 +1,43 @@
-import React, { Component } from 'react';
+import React, { FC, useContext } from 'react';
 
 import List from '@material-ui/core/List';
 
-import { VersionPageConsumerProps, DetailContextConsumer } from '../../pages/Version';
+import { DetailContext } from '../../pages/Version';
 import { Heading, DistListItem, DistChips } from './styles';
 import fileSizeSI from '../../utils/file-size';
-import { PackageMetaInterface } from 'types/packageMeta';
 import { formatLicense } from '../../utils/package';
 
-class Dist extends Component {
-  public render(): JSX.Element {
-    return (
-      <DetailContextConsumer>
-        {(context: Partial<VersionPageConsumerProps>) => {
-          return context && context.packageMeta && this.renderDist(context.packageMeta);
-        }}
-      </DetailContextConsumer>
-    );
-  }
-
-  private renderChips(dist, license: PackageMetaInterface['latest']['license']): (JSX.Element | undefined)[] {
-    const distDict = {
-      'file-count': dist.fileCount,
-      size: dist.unpackedSize && fileSizeSI(dist.unpackedSize),
-      license,
-    };
-
-    const chipsList = Object.keys(distDict).map((dist, key) => {
-      if (!distDict[dist]) return;
```

Comment:
Not sure if this is a bug or intentional, but this will hide the chips for file count or size if they happen to be 0. I've assumed it's intentional and included the same behavior.